lib/qbo_api/connection.rb

```diff
@@ -38,6 +42,7 @@ def build_connection(url, headers: nil)
     end
 
     def request(method, path:, entity: nil, payload: nil, params: nil, headers: nil)
+      # TODO: there is a #raw_response method in Faraday::Response::Json
```

Comment:
What do you think you'd do with it?

Reply:
46 would just be an attribute on the Response object

Reply:
I'd vote to leave it as is. Maybe in the future. 

However, is there is a solid reason/use case for exposing `Faraday::Response::Json.raw_response`?

Reply:
Not that I can think of, I just saw it while looking through Faraday code and thought this was duplicated code, but I could be wrong. I don't have a whole lot of context on this gem. I'm just trying to get our dependencies sorted out :)

Reply:
Going to remove this comment.